pkg/consts/views.go

```diff
@@ -55,6 +55,19 @@ function(doc) {
 }`,
 }
 
+// FilesByParentView is the view used for fetching files referenced by a
+// given document
+// TODO change me for flag hidden
+var FilesByParentView = &couchdb.View{
+	Name:    "by-parent-type-name",
```

Comment:
I also think it would be better to use the view for iterating too. Maybe a TODO and an issue to not forget.